test/index.spec.js

```diff
@@ -261,6 +286,10 @@ describe("cardano-metadata-submitter", () => {
 function fixture(done) {
   mkdtemp(path.join(os.tmpdir(), "cardano-metadata-submitter"))
     .then(cwd => {
+      copyTestData = function copyTestData() {
+        copySync(path.join(__dirname, "testData"), path.join(cwd, "testData"))
```

Comment:
What is `testData` ?

Reply:
A folder that currently has only "icon.png" file for testing `--logo` option...

Reply:
Shouldn't it be committed ?

Reply:
(ah it is at the end, I missed it!)